register_forte_options.py

```diff
@@ -919,6 +926,8 @@ def register_casscf_options(options):
     options.add_bool("CASSCF_DEBUG_PRINTING", False,
                      "Enable debug printing if True")
 
+    options.add_bool("CASSCF_DONT_ORBITAL", False, "No orbital optimization if true")
```

Comment:
maybe use something that better explains what this does as:
```suggestion
    options.add_bool("CASSCF_NO_ORBOPT", False, "No orbital optimization if true")
```

Reply:
Changes made.